src/utils/types.ts

```diff
@@ -77,6 +77,226 @@ export abstract class APIBase {
     }
 }
 
+export class RESTAPI extends APIBase {
+    protected contentType:string;
+    protected acceptType:string;
+
+    get = async (baseurl?:string, contentType?:string, acceptType?:string):Promise<RequestResponseData> => {
+        let ep:string = baseurl ? baseurl : this.baseurl;
+
+        // Set contentType if exists
+        if(contentType) {
+            this.contentType = contentType;
+        }
+
+        // Set acceptType if exists
+        if(acceptType) {
+            this.acceptType = acceptType;
```

Comment:
the get function shouldn't overwrite the default value defined in the function. This error is in post, put, delete, patch as well.